source/dotnet/wholesale-api/CalculationResults/CalculationResults.Infrastructure/CalculationResults/CalculationResultQueries.cs

```diff
@@ -54,9 +54,10 @@ await foreach (var p in GetInternalAsync(sql, batch.PeriodStart.ToInstant(), bat
 
     public async IAsyncEnumerable<CalculationResult> GetAsync(CalculationResultQuery query)
     {
-        var sql = CreateRequestSql(query);
-        await foreach (var p in GetInternalAsync(sql, query.StartOfPeriod, query.EndOfPeriod))
+        var sqlStatement = CreateRequestSql(query);
+        await foreach (var p in GetInternalAsync(sqlStatement, query.StartOfPeriod, query.EndOfPeriod))
             yield return p;
```

Comment:
What is `p`? How about this?
```suggestion
        await foreach (var result in GetInternalAsync(sqlStatement, query.StartOfPeriod, query.EndOfPeriod))
            yield return result;
```